extension/src/format/sexpression.ts

```diff
@@ -1145,13 +1145,13 @@ export class Sexpression extends LispAtom {
             gIndentSpaces = 6;
 
         let parenStyle = closeParenStyle();
-        if (parenStyle.toString() == "Same line")
+        if (parenStyle.toString().toLocaleLowerCase() == "same line")
```

Comment:
Is there a special reason for toLocaleLowerCase() ? according to the old code, the left operator should be an English string.

Reply:
This is a mistake, it should be .tolowercase()